datashader/glyphs.py

```diff
@@ -32,25 +31,49 @@ def validate(self, in_dshape):
     @staticmethod
     @ngjit
     def _compute_x_bounds(xs):
+        if len(xs) == 0:
+            raise ValueError('x coordinate array is empty.')
```

Comment:
Computing len() for a Dask dataframe is non-trivial, since it is loaded lazily, but I think this entire branch can be avoided by simply setting minval=maxval=np.NaN.